projects/openvswitch/json_parser_target.c

```diff
@@ -0,0 +1,70 @@
+#include "openvswitch/json.h"
+#include "jsonrpc.h"
+#include <string.h>
+#include <assert.h>
+#include "ovsdb-error.h"
+#include "ovsdb/table.h"
+
+static void
+print_and_free_json(struct json *json)
+{
+    char *string = json_to_string(json, JSSF_SORT);
+    json_destroy(json);
+    puts(string);
+    free(string);
+}
+
+int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
+{
+    if ((size == 0) || (data[size-1] != '\0')) return 0;
+
+    struct json *j1,*j2;
+    struct jsonrpc_msg *msg;
+//    struct ovsdb_table_schema *ts;
```

Comment:
We don't like committing commented out code to our repo. 
If the code is useful for debugging, then `#ifdef` it out during OSS-Fuzz builds, if not please remove.
Alternatively since fuzz targets would be in the project's actual repo you can commit it there and put whatever you like in the target.

Reply:
Ack.